sspl_test/plans/self.pln

```diff
@@ -0,0 +1,4 @@
+alerts.self_hw.test_self_hw_real_stor_enclosure_conn
```

Comment:
Ensure that node_id that from `from alerts.self_hw.self_hw_utilities import get_node_id` is only from GLOBAL_CONF.
There it is hard code to 'srvnode-1'.

Reply:
Note: Also, we are advised to avoid direct salt commands.

Reply:
Done, Instead of salt cmd, used  conf store api to read node_key value , in `"from alerts.self_hw.self_hw_utilities import get_node_id".`